languages/fr/modules.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * Extension for Contao Open Source CMS
+ *
+ * Copyright (C) 2013 terminal42 gmbh
+ *
+ * @package    TableLookupWizard
+ * @link       http://www.terminal42.ch
+ * @license    http://opensource.org/licenses/lgpl-3.0.html LGPL
+ */
+
+
+/**
+ * Extension folder
+ */
+$GLOBALS['TL_LANG']['MOD']['tablelookupwizard'] = array('Assistant de recherche de table');
```

Comment:
I do have a doubt for this one.
Not sure if you want to translate 'table' like a 'table' or like an 'entry'/'record'.

Reply:
It's only shown in the system settings anyway. Nothing an end user will ever see so it may remain a technical term :)